src/power_bounded.lean

```diff
@@ -28,7 +28,7 @@ begin
   split, {exact hU},
   intros v hv b H,
   cases H with n H,
-  induction n; simpa [H.symm, pow_succ, mem_of_nhds hU] 
+  induction n ; { simp [H.symm, pow_succ, mem_of_nhds hU], try {assumption} }
```

Comment:
What is wrong with `simpa`?

Reply:
Same answer as above: it didn't compile.

Reply:
`simpa` is not the same as "simp;assumption", although nobody updates the docs to indicate this. You can find Mario explaining it several times in Zulip now.